crossplane/install-crossplane.hbs.md

```diff
@@ -3,6 +3,8 @@
 This topic tells you how to install Crossplane from the Tanzu Application Platform package
 repository.
 
+> **Note** If installing Tanzu Application Platform to a cluster which already has a pre-existing installation of Crossplane, please refer to [Use a pre-existing or out-of-band installation of Crossplane with Tanzu Application Platform](how-to-guides/use-pre-existing-crossplane-tap.hbs.md)
```

Comment:
Would we want to add a note on the `PackageInstall` reconciliation error for cases where the user missed this note?

Reply:
I guess there's already a high-level note on the reconciliation error in the release notes. But that is a bit hidden and maybe not specific enough.

Reply:
> Would we want to add a note on the PackageInstall reconciliation error

Not sure exactly what you mean? Do you mean to improve the error message displayed on the reconcile error?

Reply:
Nope, just telling the reader what error to expect in that case...

The`PackageInstall` for the `crossplane.tanzu.vmware.com` package will fail to reconcile with the following error: `Resource already exists. Consider excluding package in tap-values`.

Reply:
Gotcha makes sense - I'll add that in.